pointcloud/src/main/scala/geotrellis/pointcloud/spark/triangulation/BoundaryDelaunay.scala

```diff
@@ -383,17 +454,27 @@ object BoundaryDelaunay {
       val newBound: ResultEdge = copyConvertBoundingLoop()
       var e = dt.boundary
       var ne = newBound
+      //navigateThis(ne)
+
       val boundingTris = collection.mutable.Set.empty[Int]
       do {
         // println(s"in CCBT $e")
+        assert(getDest(e) == halfEdgeTable.getDest(ne) && getSrc(e) == halfEdgeTable.getSrc(ne))
+        // if (circumcircleLeavesExtent(boundingExtent)(getFlip(e))) {
+        //   val f = getFlip(e)
+        //   println(s"Triangle ${(getSrc(f), getDest(f), getDest(getNext(f)), getDest(getNext(getNext(f))))} has circumcircle outside extent")
+        // }
         recursiveAddTris(getFlip(e), ne)
+        //navigateThis(ne)
         e = getNext(e)
         ne = halfEdgeTable.getNext(ne)
       } while (e != dt.boundary)
 
+      writeWKT("bounds.wkt")
       //isMeshValid(triangles, halfEdgeTable)
 
       fillInnerLoop
+      writeWKT("filled.wkt")
```

Comment:
They shouldn't be there, no.  And there are many comments and unclean sections of code that need to be touched up.  However, we can do this when we break out the Delaunay stuff from milestone/pointcloud into its own PR against master.  That code can be taken to a finished state prior to merging.